controllers/main_controller.py

```diff
@@ -111,10 +112,13 @@ def get_issue_parameters(self, issue):
 
     def refresh_issue_list_widget(self, error):
         if error:
+            self.error_messages_count += 1
+            if self.error_messages_count <= 1:
```

Comment:
Could error_messages_count be less then 1 if before it we added one point to it?